civihr_default_theme/assets/sass/partials/_panel.scss

```diff
@@ -3,14 +3,27 @@
 
 .panel {
     @include box-shadow($box-shadow-global);
+
+    &.panel--sliding-body {
+      overflow: hidden !important;
+
+      .panel-body-wrap {
+        max-height: 0;
+      }
+    }
+}
+
+.panel-body-wrap {
+  transition: max-height .4s linear;
+  max-height: 162px;
```

Comment:
It's just a value higher than the `.panel-body-wrap` original height, it's used to work with CSS transition.